code/modules/mob/living/carbon/human/species_types/diona.dm

```diff
@@ -0,0 +1,48 @@
+/datum/species/dionae
+	name = "Dionae"
+	id = "diona"
+	blacklisted = FALSE
+	sexes = FALSE
+	say_mod = "rustles"
+	default_color = "228822"
+
+
+	inherent_biotypes = list(MOB_ORGANIC, MOB_HUMANOID)
+	mutant_bodyparts = list("diona_body_markings")
+	default_features = list("diona_body_markings" = "Default")
+
+	mutanttongue = /obj/item/organ/tongue/diona
+
+	meat = /obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/lizard
+	skinned_type = /obj/item/stack/sheet/animalhide/lizard
```

Comment:
So, is their cortex being covered in lizard scales and their wood being meaty intended here?